app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -58,22 +58,28 @@ keywords:
   - Settlement Scheme
 ---
 
-If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial support from the UK government.
-
-Your eligibility for support will depend on your [immigration status](#check-your-immigration-status-and-eligibility-for-financial-support) and choice of subject. Most non-UK citizens without some type of settled status in the UK will not be eligible, unless they train to teach languages or physics.
-
 The information on this page is for:
 
 * non-UK citizens living in the UK
 * non-UK citizens living outside the UK
 * Irish citizens
-* refugees or asylum seekers
+
+
+Most non-UK citizens will not be eligible for financial support from the UK government to help with teacher training.  
+
+Some support may be available if: 
+
+* you train to teach languages or physics 
+
+* you have some type of settled status in the UK – for example, through the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)
+
+* you are an Irish citizen
 
 ## Teacher training fees
 
-As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs.
+As a non-UK candidate, you can expect to pay between £9,250 and about £36,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs. Most non-UK candidates will not be eligible for financial support to help with fees.
```

Comment:
can we say between £9,250 and £36,000 and remove the about? Or is it that it could be more than £36,000?

Reply:
It's not a hard and fast figure unfortunately as down to providers so I'd rather keep it vague-ish